doc/source/api/index.rst

```diff
@@ -43,6 +43,14 @@ system for reads and variants.
 .. toctree::
    references
 
+Sequence Annotations
```

Comment:
Hmm. Okay, this is probably just means that you and I (and maybe the larger we) have different colloquial definitions of "annotations" and "features". There's probably no right or wrong here, but we should get some consensus on how the terms are used. (Ditto for the use of "allele" and "variant" semi-interchangeably elsewhere in the schema.)